lib/ImportStatement.js

```diff
@@ -96,6 +102,9 @@ export default class ImportStatement {
 
     const importFunction = match.importFunction;
 
+    // Assume that an import with no assignment has side-effects.
+    const hasSideEffects = !match.assignment;
```

Comment:
Any import might have side-effects, so I'm a little worried about this naming. What do you think about changing it to something like `forSideEffectsOnly`?